### DISCUSSION TITLE
REST Api Guidelines Quality and Functionality are empty

### DISCUSSION THREAD

**Discussion by filfreire** [adidas-group/api-guidelines#88] [Q&A]
@zdne @Amzani @apidesigner 

Both
https://github.com/adidas-group/api-guidelines/tree/master/rest-api-guidelines/quality

and 
https://github.com/adidas-group/api-guidelines/tree/master/rest-api-guidelines/functionality

are completely empty in terms of description or links to other pages.

**Comment by apidesigner** [comment]
This is intended for the rendering. Please read the guidelines at GitBook(e.g., https://adidas.gitbook.io/api-guidelines/rest-api-guidelines/quality). This GH repo is the source only for GitBook renderer.
